src/System.Runtime/tests/System/Int32Tests.cs

```diff
@@ -255,9 +255,15 @@ public static IEnumerable<object[]> Parse_Valid_TestData()
             yield return new object[] { "100", NumberStyles.AllowTrailingSign, emptyNegativeFormat, 100 };
 
             // AllowLeadingWhite and AllowTrailingWhite
-            yield return new object[] { "123  ", NumberStyles.AllowLeadingWhite | NumberStyles.AllowTrailingWhite, null, 123 };
-            yield return new object[] { "  123", NumberStyles.AllowLeadingWhite | NumberStyles.AllowTrailingWhite, null, 123 };
+            yield return new object[] { "  123", NumberStyles.AllowLeadingWhite, null, 123 };
             yield return new object[] { "  123  ", NumberStyles.AllowLeadingWhite | NumberStyles.AllowTrailingWhite, null, 123 };
+            yield return new object[] { "123  ", NumberStyles.AllowTrailingWhite, null, 123 };
+            yield return new object[] { "123  \0\0", NumberStyles.AllowTrailingWhite, null, 123 };
```

Comment:
Is it interesting to have interior bulls?

Reply:
> Is it interesting to have interior nulls?

Yes, but those are invalid inputs, and that's already covered by the invalid input tests, e.g.
https://github.com/dotnet/corefx/pull/31045/files/4da4d062f2c4a69f80077eca806eee35d0a28a1c#diff-e57db10ee4f89545051b491913e641eaR368